docs/topics/appendix-using-war-files-spring-boot.adoc

```diff
@@ -0,0 +1,116 @@
+[[using-war-files-spring-boot]]
+= Building and Deploying {SpringBoot} Application WAR Files
+// Cnsider placing this inthe Spring Boot Runtime documentation instead?
+// You can Package and deploy your application as a war file
+[IMPORTANT]
+--
+Red Hat does not support packaging and deploying {SpringBoot} applications using WAR files in this release of {ProductShortName}.
```

Comment:
@inoxx03 can you update this to `{ProductName}` or update document-attributes.adoc to re-enable `ProductShortName`

other than that, LGTM.

Reply:
@rhoads-zach thanks for calling this out, I'll re-enable product short-name and update the PR.